### PR TITLE
[dotnet] - 7.0 & 6.0 EOL for noble

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Devcontainer Image**

* Dotnet

**Description**

* EOL changes for `7.0` & `6.0` for `noble-numbat` variant

**_Changelog_**

* changes to `manifest.json` made

**Checlist**

* [x] Checked that applied changes work as expected

### CODE REVIEW

File: src/dotnet/manifest.json [hunk 1]
```diff
@@ -1,16 +1,14 @@
 {
-	"version": "1.2.1",
+	"version": "1.3.0",
```
Comment (samruddhikhandale) [inline src/dotnet/manifest.json:2]: Let's revert this, for images release we create an isolated PR for version bumps.

File: src/dotnet/manifest.json [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 	"variants": [
 		"8.0-bookworm-slim",
 		"8.0-noble",
 		"8.0-jammy",
 		"7.0-bookworm-slim",
```
Comment (samruddhikhandale) [inline src/dotnet/manifest.json:7]: Shouldn't we also remove all variants for .NET 7 as it's EOL?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/dotnet/manifest.json (hunk 2 above)
Comment (goxijana3) [reply src/dotnet/manifest.json:7]: I  wiil kill all

File: src/dotnet/manifest.json [hunk 3]
```diff
@@ -4,13 +4,8 @@
 		"8.0-bookworm-slim",
 		"8.0-noble",
```
Comment (samruddhikhandale) [inline src/dotnet/manifest.json:5]: Can you update Readme file as well?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/dotnet/manifest.json (hunk 3 above)
Comment (gauravsaini04) [reply src/dotnet/manifest.json:5]: yeah done !

File: src/dotnet/manifest.json (hunk 2 above)
Comment (gauravsaini04) [reply src/dotnet/manifest.json]: yeah done !

File: src/dotnet/manifest.json (hunk 1 above)
Comment (gauravsaini04) [reply src/dotnet/manifest.json]: done

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**